inst/plumber/v1/endpoints.R

```diff
@@ -9,23 +9,25 @@ library(pipapi)
 #* Ensure that version parameter is correct
 #* @filter validate_version
 function(req, res) {
-  # tictoc::tic("filters")
-  # browser()
-  if (!is.null(req$argsQuery$version) & !grepl("swagger", req$PATH_INFO)) {
-    if (!is.null(req$argsQuery$version)) {
-      if (!req$argsQuery$version %in% lkups$versions) {
+
+  #If no arguments are passed, use the latest version
+  if (is.null(req$argsQuery$release_version) && is.null(req$argsQuery$ppp_version) &&
+     is.null(req$argsQuery$version) && is.null(req$argsQuery$identity)) {
+      version <- lkups$latest_release
+  } else {
+    if(is.null(req$argsQuery$identity)) req$argsQuery$identity <- 'PROD'
+    version <- pipapi::return_correct_version(req$argsQuery$version, req$argsQuery$release_version, req$argsQuery$ppp_version, req$argsQuery$identity, lkups$versions)
+  }
+    #If the version is not found (404) or it is not present in valid versions vector return an error.
+    if (version == "404" || !version %in% lkups$versions) {
```

Comment:
I removed `version == "404"` and left only `if (!version %in% lkups$versions)`, which should be sufficient